app/version-check/server/functions/getNewUpdates.js

```diff
@@ -11,14 +10,11 @@ import { getWorkspaceAccessToken } from '../../../cloud/server';
 export default () => {
 	try {
 		const uniqueID = Settings.findOne('uniqueID');
-		const { _oplogHandle } = MongoInternals.defaultRemoteCollectionDriver().mongo;
-		const oplogEnabled = _oplogHandle && _oplogHandle.onOplogEntry;
 
 		const params = {
 			uniqueId: uniqueID.value,
-			installedAt: uniqueID.createdAt,
+			installedAt: uniqueID.createdAt.toJSON(),
```

Comment:
This is another reason why the previous version won't work. Sadly, meteor has an issue parameterizing date objects.